chapters/welcome.ipynb

```diff
@@ -29,28 +29,69 @@
     "docker compose up\n",
     "```\n",
     "\n",
-    "The book's code examples work with many different search engines and vector databases. By default, the book uses Apache Solr as the primary storage, matching, and ranking engine, but you can easily swap in your your favorite engine by running the `set_engine` function here: [engines/supported-engines.ipynb](engines/supported-engines.ipynb).\n",
-    "\n",
-    "If you find the examples not running as expected. Please ensure you have built the entire AI-Powered Search project and are running it per the instructions at:\n",
-    "http://github.com/treygrainger/ai-powered-search\n"
+    "The book's code examples work with many different search engines and vector databases. By default, the book uses Apache Solr as the primary storage, matching, and ranking engine, but you can easily swap in your your favorite engine by running the `set_engine`"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Run Healthcheck:"
+    "### Set a search engine:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'engines.solr.SolrEngine.SolrEngine'>\n"
+     ]
+    }
+   ],
+   "source": [
+    "from aips import set_engine, get_engine\n",
+    "\n",
+    "engine_name = \"solr\"\n",
+    "set_engine(engine_name)\n",
+    "print(type(get_engine()))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Check the status of the search service stack by running a `healthcheck`. If you find the examples not running as expected. Please ensure you have built the entire AI-Powered Search project and are running it per the instructions at:\n",
+    "http://github.com/treygrainger/ai-powered-search "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run Healthcheck:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
    "metadata": {
     "scrolled": true
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "All Systems are ready. Happy Searching!\n"
```

Comment:
Need to rerun this notebook for the above reason (set_engine cell changes), as well once the healthcheck is modified elsewhere to continue printing out running services for each engine (Solr, Zookeeper, etc.), per previous comments (and how it's described in the manuscript).

Reply:
I've finished implementing the health_check logic, it was only half implemented. Notebook has been rerun